### DISCUSSION TITLE
Install Error in Script Now

### DISCUSSION THREAD

**Discussion by Aurasphere-au** [nlef/moonraker-telegram-bot#332] [General]
![image](https://github.com/user-attachments/assets/71147278-fde5-4e02-8feb-58e401891d50)

Just tried installing and all was good until I ran the script. 

?

**Comment by nlef** [comment]
Judging by the screenshot, there are no problems with bot installation scripts. There are some problems with setting up the linux  distribution repositories.

**Reply by Aurasphere-au** [reply]
Yep, the repo is no longer deemed a release

**Reply by Aurasphere-au** [reply]
No one else having probs with latest install?

**Reply by nlef** [reply]
use an internet search engine for the error text
this problem has nothing to do with the bot, you need to change the settings of your debian repositories

**Comment by Aurasphere-au** [comment]
Sure
Too much trouble
I'm not a Linux guru and if its this hard...I'll just leave it.

The issue is it doesn't work out of the box for me yet everyone else has no
issues

It sounded good :)
